mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.tsx

```diff
@@ -0,0 +1,50 @@
+import { ThemedScrollView, ThemedText } from '@components/themed'
+import { AccordionContent, WalletAccordion } from '@components/WalletAccordion'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+
+export function DexFaq (): JSX.Element {
+  const faqContent: AccordionContent[] = [
+    {
+      title: translate('components/DexFaq', 'Transaction fees'),
+      content: [{
+        text: translate('components/DexFaq', 'Each transaction will be subject to a small amount of fees. The amount may vary depending on the network\'s congestion.'),
+        type: 'paragraph'
+      }]
+    },
+    {
+      title: translate('components/DexFaq', 'What is slippage tolerance?'),
+      content: [{
+        text: translate('components/DexFaq', 'Due to the dynamic nature of the blockchain, prices on the DEX change every block. \n\nThis results in **slippage**, which is the difference between the final transaction price and the price displayed on the app before transaction confirmation. \n\nThe slippage tolerance feature allows you to indicate the maximum slippage you are willing to accept in a transaction. \n\nIf the slippage for your transaction is higher than your slippage tolerance, the transaction will not be completed and the original amount will be returned to your wallet.'),
```

Comment:
I guess the added asterisks on `**slippage**` is intended to make it bolder. Will not work, let's just remove it

Reply:
noted